index.js

```diff
@@ -1,11 +1,14 @@
 const { getOptions } = require('loader-utils');
 const postcss = require('postcss');
-const kremId = require('./src/kremling-id');
 const postcssKremling = require('./src/postcss-kremling-plugin');
 
+let globalId = 0;
```

Comment:
the `kremId` stuff was overcomplicated - just added a global Id that increments

Reply:
👍